pennylane/transforms/__init__.py

```diff
@@ -32,8 +32,6 @@
     ~transforms.classical_jacobian
     ~batch_params
     ~batch_input
-    ~draw
-    ~draw_mpl
```

Comment:
I'm not sure why I have had to remove these from recent PRs to get the docs to build both locally and for CI?

@christianbmadsen perhaps you have some insight on this? @josh146 mentioned these functions have been moved recently

Reply:
looks like my attempt to remove them this time is causing tests to fail :thinking:

Reply:
They were moved to the `drawer` module. I thought use of module code and conceptual similarity outweighed any structural similarities.  These functions were highly dependent on the `drawer` module, but nothing in the transforms module.  Nothing in the transforms module depended on them.

Reply:
it sounds like a merge conflict happened at some stage then, either locally @anthayes92 or in master

Reply:
Okay, so it sounds like they can be removed form here? 

Though it now looks like removing this is causing CI [tests to fail](https://github.com/PennyLaneAI/pennylane/runs/5859066500?check_suite_focus=true)